util/opentelemetry-util-genai-traceloop-translator/src/opentelemetry/util/genai/traceloop/__init__.py

```diff
@@ -0,0 +1,210 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import logging
+import os
+from typing import Any, Dict
+
+from opentelemetry import trace
+
+_ENV_DISABLE = "OTEL_GENAI_DISABLE_TRACELOOP_TRANSLATOR"
+_LOGGER = logging.getLogger(__name__)
+
+# Default attribute transformation mappings (traceloop.* -> gen_ai.*)
+# This is the single source of truth for all attribute mappings.
+#
+# Mapping Status Legend:
+# ✅ OFFICIAL: Documented in OpenTelemetry GenAI semantic conventions
+# ⚠️  CUSTOM: Traceloop-specific, not in official semconv (extension attributes)
+_DEFAULT_ATTR_TRANSFORMATIONS = {
+    "rename": {
+        # Content mappings (special handling with normalization)
+        "traceloop.entity.input": "gen_ai.input.messages",  # OFFICIAL
+        "traceloop.entity.output": "gen_ai.output.messages",  # OFFICIAL
+        # Agent and conversation
+        "traceloop.entity.name": "gen_ai.agent.name",  # OFFICIAL (agent spans)
+        "traceloop.correlation.id": "gen_ai.conversation.id",  # OFFICIAL (Conditionally Required)
+        # ⚠️  CUSTOM: Workflow / entity hierarchy (Traceloop-specific extensions)
+        "traceloop.workflow.name": "gen_ai.workflow.name",
+        "traceloop.entity.path": "gen_ai.workflow.path",
+        "traceloop.association.properties": "gen_ai.association.properties",
+        "traceloop.entity.version": "gen_ai.workflow.version",
+        "traceloop.span.kind": "gen_ai.span.kind",
+    }
```

Comment:
FYI, here's the current traceloop -> sem conv mapping. we may need to delete few fields that can't be translated as-is.